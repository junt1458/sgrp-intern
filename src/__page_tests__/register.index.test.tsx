import { render } from '@testing-library/react';
import RegisterIndexPage from '../pages/register';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/',
      push: jest.fn(),
    };
  },
}));

it('should render the component', () => {
  const { container } = render(<RegisterIndexPage />);
  expect(container).toMatchSnapshot();
});
