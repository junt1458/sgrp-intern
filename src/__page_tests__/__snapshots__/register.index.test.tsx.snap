// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component 1`] = `
<div>
  <div
    class="flex h-14 w-full items-center justify-between bg-slate-900 px-4 py-2 text-xl"
  >
    Internship Application System
  </div>
  <div
    class="my-4"
  >
    <h1
      class="text-center text-3xl"
    >
      No account found in our system.
    </h1>
    <h2
      class="text-center text-xl"
    >
      Please enter registration code provided by administrator.
    </h2>
  </div>
  <div
    class="my-8 flex items-center justify-center"
  >
    <input
      class="mx-2 rounded-lg px-4 py-2 text-black"
      id="reg_code"
      placeholder="Registration Code"
      type="text"
    />
    <button
      class="rounded-lg px-4 py-2"
      style="background-color: rgb(24, 168, 217); color: rgb(0, 0, 0);"
    >
      Submit
    </button>
  </div>
  <div
    class="text-center"
  />
</div>
`;
